src/inference/src/system_conf.cpp

```diff
@@ -75,6 +75,11 @@ bool with_cpu_x86_avx2_vnni() {
     return get_cpu_info().has(Xbyak::util::Cpu::tAVX2 | Xbyak::util::Cpu::tAVX_VNNI);
 }
 
+bool with_cpu_x86_avx2_vnni_2() {
+    return with_cpu_x86_avx2_vnni() && get_cpu_info().has(Xbyak::util::Cpu::tAVX_VNNI_INT8) &&
+           get_cpu_info().has(Xbyak::util::Cpu::tAVX_NE_CONVERT);
+}
+
```

Comment:
I would propose to avoid adding such function in public API given the fact there is no official ISA called "avx2_vnni_2". Lets make this function local for CPU test utils.